INSTALL.md

```diff
@@ -250,3 +251,14 @@ To test that the build works, you can try running the test suite, and a one-line
 ./tests/bpftrace_test
 ./src/bpftrace -e 'kprobe:do_nanosleep { printf("sleep by %s\n", comm); }'
 ```
+
+# Disable Lockdown
+
```

Comment:
not sure what to put here, other than disabling secure boot in the bios

Reply:
> not sure what to put here, other than disabling secure boot in the bios

Disabling secure boot in the bios is one option. Another option is `sudo mokutil --disable-validation`, rebooting, and then following the prompts to disable secure boot verification in `shim`.